PlanningPoker/ClientApp/src/app/card/card.component.ts

```diff
@@ -18,17 +18,17 @@ export class CardComponent implements OnInit {
 
   getHeaderClass() {
 
-    return localStorage.getItem('UserVote') === this.value.toString() ? 'card border-success  mb-3 bg-success' :
+    return sessionStorage.getItem('UserVote') === this.value.toString() ? 'card border-success  mb-3 bg-success' :
```

Comment:
You can create a helper method for sessionStorage.getItem('UserVote') === this.value.toString() and reuse it in two places